test/pet.test.js

```diff
@@ -0,0 +1,189 @@
+const Pet = require('../src/pet');
+
+let testPet;
+
+const deadMessage = "Your pet is no longer alive :("
+
+beforeEach(() => {
+    testPet = new Pet();
+  });
+
+describe('constructor', () => {
+    it('should return a new pet with a name', () => {
+        expect(new Pet()).toBeInstanceOf(Object);
+        expect(new Pet().name).toBe("An unloved pet");
+        expect(new Pet('Manny').name).toBe("Manny");
+    })
+
+    it('should return a new pet with an age', () => {
+        expect(new Pet().age).toBe(0);
+    })
+
+    it('shold return a new pet with a hunger', () => {
+        expect(new Pet().hunger).toBe(0);
+    })
+
+    it('shold return a new pet with a fitness', () => {
+        expect(new Pet().fitness).toBe(10);
+    })
+})
+
+describe('growUp', () => {
+    it('should make the pet grow', () => {
+        testPet.growUp();
+        expect(testPet.age).toBe(1);
```

Comment:
you can also write multiple expects in the same it block, if you wanted to test it wasn't just fluke the first time.